vmss-linux-nat/azuredeploy.json

```diff
@@ -0,0 +1,281 @@
+{
+  "$schema":"http://schema.management.azure.com/schemas/2015-01-01-preview/deploymentTemplate.json",
+  "contentVersion":"1.0.0.0",
+  "parameters":{
+    "resourceLocation":{
+      "type":"string",
+      "allowedValues":[
+        "East US",
+        "West US",
+        "Central US",
+        "South Central US",
+        "East US 2",
+        "North Europe",
+        "West Europe",
+        "Southeast Asia",
+        "East Asia",
+        "Japan West",
+        "Japan East"
+      ],
+      "metadata":{
+        "description":"Azure region where all resources will be deployed."
+      }
+    },
+    "vmSku":{
+      "type":"string",
+      "allowedValues":[
```

Comment:
We could remove allowed values. If we remove allowed values, does defaultValue still show up somewhere? It would be nice to give a reasonable default.